File: ch1-3.js
function unduplicator(string){
	var result = ""

	for (var i = 0; i < string.length; i++){
		var letter = string.charAt(i);

		if (result.includes(letter) === false) {
			result += letter
		}
	}
	return result
}

console.log(unduplicator("hello"))

File: ch1-3.rb
# remove duplicate characters from a string

# input: string

# for each character, store characters that have have not been used yet in the string

# output: string without duplicate characters

def unduplicater(string)
  result = ""
  string.each_char do |letter|
    if !result.include?(letter)
      result += letter
		end
  end
  return result
end

p unduplicater("aa")
p unduplicater("hello")